src/routes/repos.json.ts

```diff
@@ -1,73 +1,172 @@
 import { Octokit } from 'octokit'
 import dayjs from 'dayjs'
+import { readFileSync, existsSync } from 'fs'
+import Promise from 'bluebird'
 
-import { assertEqual, assertExists } from '$lib/assert'
+import { run } from '$lib/run'
+import { assertEqual, assertExists, assertPresent } from '$lib/assert'
 import {
   GITHUB_ACCESS_TOKEN,
   GITHUB_ORGS_ALL_REPOS,
   MAX_DAYS_SINCE_LAST_PUSH,
   MAX_REPOS,
+  UPDATE_EXISTING_REPOS,
 } from '$lib/env'
 
+const REPOS_DIR = `${process.cwd()}/repos`
+const HOLOCHAIN_REPO_NAME = `holochain/holochain`
+const OCTOKIT = new Octokit({ auth: GITHUB_ACCESS_TOKEN })
+
 // eslint-disable-next-line @typescript-eslint/ban-types
-export async function get(): Promise<{ body: { repos: Array<object> } }> {
-  const octokit = new Octokit({ auth: GITHUB_ACCESS_TOKEN })
+export async function get(): Promise<{ body: { repos: Array<RepoForUi> } }> {
```

Comment:
Refactored get() into high level function calls; before it was one big function.